pkg/metrics/metrics.go

```diff
@@ -89,9 +89,25 @@ var (
 			NodePoolLabel,
 		},
 	)
+	IgnoredPodCount = prometheus.NewGauge(
```

Comment:
This is only relevant to provisioning -- should this be in that package's metrics file